packages/client/src/Components/ChatChannel.tsx

```diff
@@ -88,8 +87,8 @@ const ChatChannel: React.FC<{ channelId: string, isCustomChannel?: boolean }> =
   return (
     <div className={channelTabClass} data-channel-id={channelId}>
       {
-        typeof isCustomChannel === 'boolean' && isCustomChannel
-          ? <ChannelMessagesList 
+        typeof isCustomChannel === 'boolean' && !isCustomChannel
```

Comment:
this new `!` looks scary.  I'm pretty sure `ChannelMessagesList` is the component to use for `isCustomChannel`.  

Will double-check code when I get chance.

Reply:
sorry @IanMayo , it's a typo, I updated